example/idp/main.go

```diff
@@ -148,12 +166,33 @@ func main() {
 	hydraConfig := helpers.NewHydraConfig(*configPath)
 
 	// Setup the providers
-	var err error
-	provider, err = basic.NewBasicAuth(*htpasswdPath, "localhost")
+	userdb, err := memory.NewMemStore()
+	if err != nil {
+		panic(err)
+	}
+
+	err = userdb.LoadHtpasswd(*htpasswdPath)
 	if err != nil {
 		panic(err)
 	}
 
+	if *useForm {
```

Comment:
Sure, no problem.  That probably in fact gives me a little more freedom with a few things.